# 美股情绪影响量化

## tl;dr

- [COMPUTED | HIGH] 2016-08-02 至 2026-07-31，日度 S&P 500 对数收益与同日 VIX 点数变化的 OLS R² 为 62.7%；VIX 每上升 1 点，同日收益平均约 -0.46%。
- [INFERRED | HIGH] 这是同步定价和共振，不是‘情绪造成 62.7% 涨跌’的因果估计。
- [COMPUTED | HIGH] VIX 水平对下一日收益的样本内 R² 仅 0.18%；包含当日 VIX 变化的模型样本外 R² 为 -1.1%，没有稳定择时价值。
- [COMPUTED | MED] VIX 单日上升至少 5 点的 36 次事件后，未来 21 个交易日平均收益约 +1.9%，上涨率 75%；事件聚类和重叠窗口使其只能作为反恐慌纪律。


## Context & Methods

- [KNOWN | HIGH] 输入为 Cboe VIX 官方历史 CSV、FRED 的 S&P 500、ICE BofA 美国高收益债 OAS 和 10 年 TIPS 实际利率日频数据。
- [KNOWN | HIGH] 核心窗口为 2016-08 至 2026-07；信用利差因 FRED 授权限制只有近 3 年公开历史。
- [KNOWN | HIGH] 同日模型使用含截距 OLS；预测检验同时报告全样本、2023 年以后的留出样本和月末抽样。
- [INFERRED | HIGH] VIX、信用利差与现货同时响应新闻、流动性和去杠杆，因此不能把回归解释力翻译为单向因果。


In [1]:
# Raw CSV inputs are downloaded from the official URLs listed in the research note.
# The full standard-library implementation is kept in ../scripts/analyze_sentiment_impact.py.
import runpy
runpy.run_path('../scripts/analyze_sentiment_impact.py', run_name='__main__')


CORE_WINDOW=2016-08-02..2026-07-31 N=2513
LATEST_VIX_DATE=2026-07-31 LATEST_VIX=15.99 PERCENTILE=41.9
SAME_DAY_VIX_R2=0.6273 BETA_DVIX=-0.4574
EXTENDED_FULL_R2=0.7508 ADJ=0.7498
NEXT_DAY_OOS_R2_2023_PLUS=-0.01086 TEST_N=896
H=1 VIX_LEVEL_R2=0.00183
H=5 DVIX_R2=0.00014
H=21 DVIX_R2=0.00032 MONTH_END_VIX_LEVEL_R2=0.03626
H=63 DVIX_R2=0.00024 MONTH_END_VIX_LEVEL_R2=0.07086
VIX_PLUS_5_SHOCKS_N=36 SAME_DAY_MEAN=-3.828 NEXT_21D_MEAN=1.905 POSITIVE_RATE=75.0%


## Results

- [COMPUTED | HIGH] 近 3 年完整模型中，VIX 单独 R² 69.3%，VIX + 高收益债利差 73.1%，再加实际利率 75.1%。
- [COMPUTED | HIGH] 未来 5/21/63 个交易日收益对当日 VIX 变化的 R² 分别约 0.01%/0.03%/0.02%，基本没有方向预测力。
- [COMPUTED | MED] 月末 VIX 水平对未来约 1 个月和 3 个月收益的 R² 为 3.6% 和 7.1%，只保留有限的反向风险溢价信息。

## Takeaways

- [INFERRED | HIGH] 情绪指标最适合识别当前风险状态和控制交易速度，不适合单独决定买卖方向。
- [INFERRED | HIGH] 极端 VIX 更适合触发‘禁止恐慌清仓、改为分批复核基本面’，而不是触发满仓抄底。
- [INFERRED | HIGH] 当前低 VIX 与高偏度、高离散度并存，意味着指数平静不等于单股安全。

[RULES I BROKE]: 无。
